index.html

```diff
@@ -232,5 +232,42 @@ <h3>Skills</h3>
             </div>
         </div>
     </section>
+    <br>
+    <section>
+        <!-- Contact form -->
+        <div class="div4">
+            <h2 class="color1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+        <br>
+        <div class="div5">
+            <form id="contact-form" action="https://formspree.io/f/mlevonav" method="POST">
+                <div>
+                    <input class="input-fields" type="text" name="name" id="name" placeholder="Full Name" max="30" required>
+                </div>
+                <div>
+                    <input class="input-fields" type="email" name="email" id="email" placeholder="Email Address" required>
+                </div>
+                <div>
+                    <textarea class="input-textarea" name="message" id="message" placeholder="Enter text here" cols="20" rows="10" maxlength="500" required></textarea>
+                </div>
+                <div class="title3">
+                    <button class="seeproject" type="submit">
+                        Get in touch
+                    </button>
+                </div>
+                <ul class="socials2">
+                    <li><a href="#"><img src="images/SVG/twitter.svg" alt="twitter-icon"></a></li>
+                    <li><a href="#"><img src="images/SVG/Linkedin icon.svg" alt="linkedin-icon"></a></li>
+                    <li><a href="#"><img src="images/SVG/medium.svg" alt="medium-icon"></a></li>
+                    <li><a href="#"><img src="images/SVG/github.svg" alt="github-icon"></a></li>
+                    <li><a href="#"><img src="images/SVG/angellist.svg" alt="angellist-icon"></a></li>
+                </ul>
```

Comment:
- [ ] **[OPTIONAL]:** I would suggest moving this social media list out of the `<form>` tag. 📜  This will provide a better semantic meaning to your document since they aren't related. It would be great if you change this! 🤓 🛠️